include/ada/serializers.h

```diff
@@ -5,11 +5,8 @@
 #ifndef ADA_SERIALIZERS_H
 #define ADA_SERIALIZERS_H
 
-#include "ada/common_defs.h"
-
-#include <array>
-#include <optional>
 #include <string>
```

Comment:
```suggestion
#include <array>
#include <string>
```

You somehow delete headers... :-(